Content.Client/GameObjects/Components/Instruments/InstrumentComponent.cs

```diff
@@ -164,7 +167,8 @@ public bool CloseMidi()
         /// <param name="midiEvent">The received midi event</param>
         private void RendererOnMidiEvent(MidiEvent midiEvent)
         {
-            SendNetworkMessage(new InstrumentMidiEventMessage(midiEvent));
+            SendNetworkMessage(new InstrumentMidiEventMessage(midiEvent,
```

Comment:
Send the offset relative to `IGameTiming.CurTime` instead.

Reply:
Done!